integration/kubernetes/nginx.bats

```diff
@@ -18,19 +18,22 @@
 load "${BATS_TEST_DIRNAME}/lib.sh"
 
 setup() {
+	service_name="nginx"
+	image_name="$service_name"
```

Comment:
Why are you using 2 vars for the same value?

Reply:
changed the value of the service to avoid confusion.